<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/iLogos/logo_novafct.png" width="200">

# Departamento de Engenharia Mecânica e Industrial
## Mecânica Aplicada I 

## CENTROIDES E CENTROS DE GRAVIDADE

### Problema 2

Considere o tubo de alumínio ABCDE da figura.  Sabendo que $L = 2$ m, determine a distância $d$ de forma que a parte BCD fique horizontal quando se suspende o tubo pelo ponto C.

<img src="https://userweb.fct.unl.pt/~jmc.xavier//MAI/Notebooks/Ch05/P2/MAI_Ch05_P2.jpg"
width="500">


### Resolução

**Diagrama de corpo livre**

Começando por ilustrar o DCL na figura abaixo:

<img src="https://userweb.fct.unl.pt/~jmc.xavier//MAI/Notebooks/Ch05/P2/MAI_Ch05_P2a.jpg"
width="1000">

**Condições de equilíbrio**

Estabelecendo as equações escalares de equilíbrio do corpo rígido no plano obtém-se

\begin{equation*}
\begin{cases}
\sum F_y &= 0 \\
\sum M_\textrm{B} &= 0
\end{cases}~\Leftrightarrow~
\begin{cases}
F - P = 0 \\
Fd - Pd_1 = 0
\end{cases}~\Leftrightarrow~
\begin{cases}
F = P \\
d = d_1
\end{cases}
\end{equation*}

Verifica-se que $d = d_1$ pelo que, as forças $F$ e $P$ têm como linha de acção a linha vertical que passa pelos Pontos $C$ e $G$, ou seja, conforme se ilustra pela figura abaixo

<img src="https://userweb.fct.unl.pt/~jmc.xavier//MAI/Notebooks/Ch05/P2/MAI_Ch05_P2b.jpg"
width="500">



**Cálculo do centro de gravidade de um fio composto homogéneo**

Para o cálculo do centro de gravidade do tubo de alumínio ABCDE, usam-se as expressões de um fio  composto por diferentes troços homogéneo conforme a figura abaixo, recorre-se às expressões (sistema de coordenadas escolhido tem origem no ponto da secção B):

<img src="https://userweb.fct.unl.pt/~jmc.xavier//MAI/Notebooks/Ch05/P2/MAI_Ch05_P2c.jpg"
width="600">

\begin{equation*}
\begin{cases}
\overline{x} &=
\displaystyle\frac{\sum_{i=1}^3\overline{x}_i\cdot
L_i}{\sum_{i=1}^3 L_i}
= \displaystyle\frac{\overline{x}_\textrm{AB}L_\textrm{AB} +
\overline{x}_\textrm{BD}L_\textrm{BD} +
\overline{x}_\textrm{DE}L_\textrm{DE}}{L_\textrm{AB} + L_\textrm{BD} +
L_\textrm{DE}}
\\
\overline{y} &= \displaystyle\frac{\sum_{i=1}^3\overline{y}_i\cdot
L_i}{\sum_{i=1}^3 L_i}
= \displaystyle\frac{\overline{y}_\textrm{AB}L_\textrm{AB} +
\overline{y}_\textrm{BD}L_\textrm{BD} +
\overline{y}_\textrm{DE}L_\textrm{DE}}{L_\textrm{AB} + L_\textrm{BD} +
L_\textrm{DE}}
\end{cases}
\end{equation*}

_TROÇO AB_

\begin{equation*}
\overline{x}_\textrm{AB} = \displaystyle\frac{L_\textrm{AB}}{2}\cos
(\measuredangle ABD)
\quad\wedge\quad
\overline{y}_\textrm{AB} = -\frac{L_\textrm{AB}}{2}\sin(\measuredangle
ABD)
\quad\wedge\quad L_\textrm{AB} = 0.75~\textrm{m}
\quad\wedge\quad
\measuredangle ABD = 55^\circ
\end{equation*}

numericamente,

\begin{equation*}
\overline{x}_\textrm{AB} =
\frac{0.75}{2}\cos(55^\circ\frac{\pi}{180^\circ})
\quad\wedge\quad
\overline{y}_\textrm{AB} = -\frac{0.75}{2}\sin
(55^\circ\frac{\pi}{180^\circ})
\end{equation*}

In [12]:
import numpy as np
BG = 0.75 # unit: m
ang55 = np.deg2rad(55) # unit: radianos
xBG = BG/2*np.cos(ang55)
print('xAB = ','{:.3f}'.format(xBG),' m')
yBG = -BG/2*np.sin(ang55)
print('yAB = ','{:.3f}'.format(yBG),' m')

xAB =  0.215  m
yAB =  -0.307  m


_TROÇO BD_

\begin{equation*}
\overline{x}_\textrm{BD} = \displaystyle\frac{L_\textrm{BD}}{2} = \displaystyle\frac{1.5}{2}
\quad\wedge\quad
\overline{y}_\textrm{BD} = 0
\quad\wedge\quad L_\textrm{BD} = 1.5~\textrm{m}
\end{equation*}

In [13]:
BD = 1.5 # unit: m
xBD = BD/2
print('xBD = ','{:.3f}'.format(xBD),' m')
yBD = 0
print('yBD = ','{:.3f}'.format(yBD),' m')

xBD =  0.750  m
yBD =  0.000  m


_TROÇO DE_

\begin{equation*}
\begin{cases}
\overline{x}_\textrm{DE} &= L_\textrm{BD} - \displaystyle\frac{L_\textrm{DE}}{2}
\cos(\measuredangle BDE)
\quad\wedge\quad
\overline{y}_\textrm{DE} = -
\displaystyle\frac{L_\textrm{DE}}{2}\sin(\measuredangle BDE)
\quad\wedge\quad L_\textrm{DE} \equiv L = 2.0~\textrm{m}\\
\overline{x}_\textrm{DE} &= 1.5 -
\displaystyle\frac{2}{2}\cos(55^\circ\frac{\pi}{180^\circ})
\quad\wedge\quad
\overline{y}_\textrm{DE} = -\displaystyle\frac{2}{2}\sin(55^\circ\frac{\pi}{180^\circ})
\end{cases}
\end{equation*}

In [14]:
DE = 2 # unit: m
xDE = BD - DE/2*np.cos(ang55)
print('xDE = ','{:.3f}'.format(xDE),' m')
yDE = -DE/2*np.sin(ang55)
print('yDE = ','{:.3f}'.format(yDE),' m')

xDE =  0.926  m
yDE =  -0.819  m


|Troço|$L_\textrm{i}$,m|$x_\textrm{i}$,m|$y_\textrm{i}$,m|$x_\textrm{i}L_\textrm{i}$,m $^2$|$y_\textrm{i}L_\textrm{i}$, m $^2$|
|:---:|:--------------:|:-------------------------:|:-------------------------:|:-------------------:|:--------------------:|
| BG | 0.75 | 0.215 | -0.307 | 0.161 | -0.230 |
| BD | 1.5 | 0.75 | 0.0 | 1.125  | 0.0 |
| DE | 2 | 0.926 | -0.819 | 1.852 | -1.640 |
| $\sum$ | 4.25 | - | - | 3.139  | -1.869 |

In [20]:
print('Comprimento total  : ------- ')
Lt = BG + BD + DE
print('Lt = ','{:.2f}'.format(Lt),' m')

print('Centroides : ------- ')
xL = xBG*BG + xBD*BD + xDE*DE
print('sum xiLi = ','{:.3f}'.format(xL),' m2')
X_centr = xL/Lt
print('X_centr = ','{:.3f}'.format(X_centr),' m')
yL = yBG*BG + yBD*BD + yDE*DE
print('sum yiLi = ','{:.3f}'.format(yL),' m2')
Y_centr = yL/Lt
print('Y_centr = ','{:.3f}'.format(Y_centr),' m')


Comprimento total  : ------- 
Lt =  4.25  m
Centroides : ------- 
sum xiLi =  3.139  m2
X_centr =  0.739  m
sum yiLi =  -1.869  m2
Y_centr =  -0.440  m


Resultando:

\begin{equation*}
\overline{x} \equiv d = 0.739~\textrm{m}
\quad\wedge\quad
\overline{y} = -0.440~\textrm{m}
\end{equation*}

---

Copyright (c) DEMI - FCT NOVA

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>